src/components/Popup.js

```diff
@@ -1,39 +1,41 @@
 class Popup {
-    constructor ({popupSelector}) {
-        this._popupElement = document.querySelector(popupSelector);
-        this._handleEscClose = this._handleEscClose.bind(this);
-        this._closeButton = this._popupElement.querySelector('.modal__close');
-        this._addEventListeners();
-    }
-
-    _addEventListeners() {
-        this._closeButton.addEventListener('click', this.close.bind(this));
-    }
+  constructor({ popupSelector }) {
+    this._popupElement = document.querySelector(popupSelector);
+    this._handleEscClose = this._handleEscClose.bind(this);
+    this._closeButton = this._popupElement.querySelector(".modal__close");
+    this._addEventListeners();
+  }
 
-    open() {
-       this._popupElement.classList.add('modal_opened');
-       document.addEventListener('keydown', this._handleEscClose);
-    }
+  _addEventListeners() {
+    this._closeButton.addEventListener("click", this.close.bind(this));
```

Comment:
Please, delete `_addEventListeners` because you have everything in `setEventListeners`


You should set the listeners only inside  `setEventListeners` according to the task of the 8th sprint